lib/msal-browser/test/interaction_client/SilentCacheClient.spec.ts

```diff
@@ -78,13 +78,14 @@ describe("SilentCacheClient", () => {
     let silentCacheClient: SilentCacheClient;
     let pca: PublicClientApplication;
 
-    beforeEach(() => {
+    beforeEach(async () => {
         pca = new PublicClientApplication({
             auth: {
                 clientId: TEST_CONFIG.MSAL_CLIENT_ID,
             },
         });
 
+        await pca.initialize();
```

Comment:
I'm curious what changed that necessitates adding this everywhere in this PR specifically...

Reply:
Where do you think msal-browser calls initialize?

Reply:
Today... standard controller is automatically there.... in this PR... it's only downloaded when needed.  That was the point of introducing the controller.  Get the controller that you need for the context you find yourself running in.  For example: you could have a WAM Controller completely separate from the non-wam code instead of all the if/else statements in the standard controller implementation.  (Meaning a much smaller download... since today you get both implementations).  I don't personally by the we may need to fallback inline... if WAM is that unreliable.... seems like there are bigger problems.